css-positioning/relative.css

```diff
@@ -0,0 +1,35 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+  position: relative;
+}
+
+#bulbasaur {
+  right: 15px;
+  top: 20px;
+}
+
+#charmander {
+  left: 30px;
+  top: 75px;
+}
+
+#squirtle {
+  left: 45px;
+  top: 200px;
+}
+
+#pikachu {
+  left: 75px;
+  top: 350px;
+}
+
+#jigglypuff {
+  left: 90px;
+  top: 400px;
+}
```

Comment:
For all pokemons, we do not have to reposition them with `left`, only `top` is needed